charts/kubearchive/templates/database/database_secret.yaml

```diff
@@ -3,8 +3,16 @@ apiVersion: v1
 kind: Secret
 type: Opaque
 metadata:
-  labels:
-    app: {{ tpl .Values.database.name . }}
   name: {{ tpl .Values.database.secret . }}
 data:
-  {{- include "kubearchive.v1.database.env" . | nindent 2 }}
+  {{- if .Values.global.production }}
+  POSTGRES_URL: ""
+  POSTGRES_DB: ""
+  POSTGRES_USER: ""
+  POSTGRES_PASSWORD: ""
+  {{- else }}
+  POSTGRES_URL: {{ .Values.database.url | b64enc | quote }}
+  POSTGRES_DB: {{ .Values.database.postgresData.dbName | b64enc | quote }}
+  POSTGRES_USER: {{ .Values.database.postgresData.dbUser | b64enc | quote }}
+  POSTGRES_PASSWORD: {{ .Values.database.postgresData.dbPassword | b64enc | quote }}
+  {{- end}}
```

Comment:
Why not only have one set of values here, and those values are the development values?  Our instructions for production will tell the user to change them, but they don't have to.  And it will always work for development.

Reply:
> Why not only have one set of values here, and those values are the development values? Our instructions for production will tell the user to change them, but they don't have to. And it will always work for development

IINM, We will address this in https://github.com/kubearchive/kubearchive/issues/380.
For now I am just leaving this logic as it was.